__tests__/endpoint.test.js

```diff
@@ -86,8 +86,18 @@ describe('GET /api/reviews/:review_id', () => {
       expect(body).toEqual({message: 'Not found'})
     })
   });
+  test('200: should add feature to also return the comment count of each review', () => {
+		return request(app)
+			.get('/api/reviews/3')
+			.expect(200)
+			.then(({ body }) => {
+				const { review } = body;
+				expect(review[0]).toHaveProperty('comment_count', '3');
```

Comment:
Ideally you want the API to send the review object alone, rather than inside an array - this can be achieved by deconstructing it inside either the model or controller instead